pallets/committee/src/lib.rs

```diff
@@ -380,6 +389,42 @@ pub mod pallet {
 
             Ok(().into())
         }
+
+        #[pallet::weight(10_000)] // TODO: Set weights
+        /// Set new constituent members
+        ///
+        /// This call can only be triggered by `fn propose_constituents` momentary
+        /// after approving of `propose_constituents`
+        pub fn set_members(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
```

Comment:
This will be the action of the constituent nominate proposal right?
Then this should ensure that the origin is approved by comittee first and then insert the new member.

Reply:
Wrapped this as a private KV set function instead a `Call`